README.md

```diff
@@ -27,18 +27,17 @@ Ensure you have the following prerequisites:
 ## Installing the Watson .NET Standard SDK
 You can get the latest SDK packages through NuGet. Installation instructions can be found in the ReadMe of each package.

```

Comment:
We should get rid of this line now that the READMEs are gone.

Reply:
Good catch - updated!